src/libs/mrubyWriterConnector.ts

```diff
@@ -492,4 +496,26 @@ export class MrubyWriterConnector {
 
     return Success.value(line);
   }
+  async verify(code: Uint8Array) {
+    const crc = calculateCrc8(code);
```

Comment:
```suggestion
    const correctHash = calculateCrc8(code);
```
`crc`はあくまで1アルゴリズムの名前です。